### ISSUE TITLE
Support `-minimal` switch for ASP.NET templates for F#

### ISSUE THREAD

**Issue by vzarytovskii** [dotnet/fsharp#13510]
Issue in ASP.NET repo: https://github.com/dotnet/aspnetcore/issues/40161

**Comment by vzarytovskii** [comment]
@charlesroddie does your vote meant you don't want to see it implemented? Or something else?

**Comment by charlesroddie** [comment]
Yes should not be implemented. Minimal apis expose non-type safe ways of doing asp.net. The type signatures are all wrong for `MapGet`. This will then infect F#. Lots of people will ask questions on forums asking for help with effectively untyped code. F# is the opposite of that and should stay miles away.

If there were a way to hide all the `MapGet` methods and only implement [this one](https://github.com/dotnet/aspnetcore/issues/40161#issuecomment-1184732028) it would be OK:
```fsharp
member this.MapGet(pattern: string, handler: Func<IResult>)
```
